### DISCUSSION TITLE
GDIObjectleak error message

### DISCUSSION THREAD

**Discussion by Gakkie79** [otya128/winevdm#1340] [Q&A]
I am running a 16bit application on a 64 bit Windows 10 laptop using OTVDM. This is an application for reading air conditioning units. I can use all functions of the application. In the application there is an option to run a live graph. However, after about 5 minutes I get an error in the application that there is not enough memory. In the Windows log I see an error message from the GDIObject leak. How can I solve this? Anyone have experience with this?
I created a trace file.
[trace_no_dll.txt](https://github.com/otya128/winevdm/files/13615618/trace_no_dll.txt)
![20231119_173748](https://github.com/otya128/winevdm/assets/153290369/f47a2063-03a7-439e-9423-9106144efd86)

**Comment by cracyc** [comment]
Really need a trace that includes the error.  It will probably be huge but trace files compress well.

**Reply by Gakkie79** [reply]
The trace file that is included as an attachment was created with the error.
[trace_no_dll.txt](https://github.com/otya128/winevdm/files/13619554/trace_no_dll.txt)

**Comment by cracyc** [comment]
Well, there isn't any sign of an object leak or the out of memory message in the trace so there isn't enough information to debug it.  I'd probably have to test the program directly.

**Reply by Gakkie79** [reply]
To test it yourself you need a Liebherr BK3 plc card. And an accompanying air conditioning unit. The application itself gives the error that there is not enough memory / system resources. Notification is in German. Exact translation is a bit difficult.

**Comment by cracyc** [comment]
Also, the version you are using is over a year old.  Try the latest from https://ci.appveyor.com/project/otya128/winevdm/builds/48555490/job/kb1aaepw9aj8me0s/artifacts .

**Reply by Gakkie79** [reply]
I think we tried it with this version too. I'm going to look into it. And possibly create a new trace file
